specifyweb/frontend/js_src/lib/components/SchemaViewer/TableList.tsx

```diff
@@ -9,29 +9,29 @@ import { SortIndicator, useSortConfig } from '../Molecules/Sorting';
 import type { SchemaViewerRow, SchemaViewerValue } from './helpers';
 
 export function SchemaViewerTableList<
-  SORT_CONFIG extends
-    | 'schemaViewerFields'
-    | 'schemaViewerRelationships'
-    | 'schemaViewerTables',
+  SORT_CONFIG extends keyof SortConfigs,
   FIELD_NAME extends SortConfigs[SORT_CONFIG],
   DATA extends SchemaViewerRow<RR<FIELD_NAME, SchemaViewerValue>>
 >({
   sortName,
+  defaultSortField,
```

Comment:
hmm why was `defaultSortField` added? it's set to the same "name" value in all the usages I see

Reply:
think I was making the entire component generic (being able to use any sort config, so needed to add that). Reverted to previous behaviour. Is kinda straightforward from here